src/main/java/thrift/model/util/CurrencyUtil.java

```diff
@@ -147,4 +121,51 @@ public static double convert(Map<String, Double> currencyMappings, double value,
         return convertFromDefaultCurrency(currencyMappings, valueInDefaultCurrency, currencyTo);
     }
 
+    /**
+     * Returns a default mapping of currency mappings
+     * @return default mapping of currency mappings
+     */
+    public static Map<String, Double> getDefaultMap() {
+        if (defaultMappings != null) {
+            return defaultMappings;
+        }
+        Map<String, Double> currencyMappings = new HashMap<String, Double>(0);
```

Comment:
Perhaps you could have this Map as a class attribute so that it is created once on initialization instead of creating a new Map each time `getDefaultMap()` is called.

Reply:
It is already a static class attribute.